xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -284,15 +275,9 @@ void CGUIDialogPVRTimerSettings::InitializeSettings()
   if (tag->SupportsFolders())
     AddEdit(group, SETTING_TMR_DIR, 19076, 0, tag->m_strDirectory, true, false, 19104);
 
-  AddToggle(group, SETTING_TMR_RADIO, 19077, 0, tag->m_bIsRadio);
```

Comment:
We many hundreds that skins use. Take weather for starters and there's way more. Splitting that out and let every skin handle it theirselves will only create a consistency nightmare